src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java

```diff
@@ -139,10 +140,8 @@ public class ExecutorStepTest {
      */
     @Test public void buildShellScriptOnSlave() throws Throwable {
         sessions.then(r -> {
-                DumbSlave s = r.createOnlineSlave();
-                s.setLabelString("remote quick");
+                DumbSlave s = r.createSlave("remote quick", null);
```

Comment:
https://javadoc.jenkins.io/component/jenkins-test-harness/org/jvnet/hudson/test/JenkinsRule.html#createSlave-java.lang.String-hudson.EnvVars- for reference